pkg/install/install_test.go

```diff
@@ -24,6 +30,22 @@ func testFillInTemplates(t *testing.T, expectedManifestCount int, params Templat
 			}
 		}
 	}
+
+	return manifests
+}
+
+func testExtractDeployment(t *testing.T, data []byte) *v1.Deployment {
```

Comment:
I would simply inline this function

Reply:
I haven't inlined this as its used in 2 different test functions.

Reply:
Ah, true. Can you rename it to `unmarshalDeployment`?